app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarView.kt

```diff
@@ -207,6 +212,18 @@ class AwesomeBarView(
         searchSuggestionProviderMap = HashMap()
     }
 
+    private fun getDefaultSearchEngine(components: Components): String? {
```

Comment:
For me this function name is misleading.  This is not getting the default search engine but the title to use with the default search engine?